### PR TITLE
Add Bingkun Pan to committers

### PR DESCRIPTION
**Pull request by panbingkun**
Add Bingkun Pan to committers.

### CODE REVIEW

File: committers.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ navigation:
 |Yin Huai|Databricks|
 |Shane Huang|Intel|
 |Dongjoon Hyun|Apple|
+|Bingkun Pan|Baidu|
```
Comment (zhengruifeng) [inline committers.md]: I think this list is sorted by last name :)

Comment (panbingkun) [reply committers.md]: Oh, I see, I was wondering about this order. 
Got it, let me update it now, thanks!

Comment (panbingkun) [reply committers.md]: Updated. ❤️

**Review by zhengruifeng — APPROVED** [review]

**Comment by zhengruifeng** [comment]
congrats!

**Comment by LuciferYang** [comment]
Congratulations ~

**Review by LuciferYang — APPROVED** [review]

**Review by yaooqinn — APPROVED** [review]

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM.

**Comment by dongjoon-hyun** [comment]
Please try to merge this PR with the merge script. 

- https://github.com/apache/spark-website/blob/asf-site/merge_pr.py

**Comment by panbingkun** [comment]
> Please try to merge this PR with the merge script.
> 
> * https://github.com/apache/spark-website/blob/asf-site/merge_pr.py

Okay, thanks!

**Review by HyukjinKwon — APPROVED** [review]
Congrats!

**Comment by panbingkun** [comment]
Merged to asf-site.
Thank you so much, @dongjoon-hyun, @zhengruifeng, @LuciferYang, @yaooqinn, @HyukjinKwon ❤️ !

**Pull request closed without merging**